torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -91,13 +101,20 @@ def __sub__(
         return self._merge(other, lambda c: -c)
 
     def transform(self, map_fn: Callable[[str], str]) -> "FunctionCounts":
+        """Apply `map_fn` to all of the function names.
```

Comment:
Also please describe what happens to counts of the functions mapped to the same name (they are added together). It can be inferred from `coalesce entries` but explicit description is always good.

Reply:
Done.